src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -46,10 +46,11 @@ var (
 	log = golog.LoggerFor("flashlight")
 
 	// Command-line Flags
-	help      = flag.Bool("help", false, "Get usage help")
-	parentPID = flag.Int("parentpid", 0, "the parent process's PID, used on Windows for killing flashlight when the parent disappears")
```

Comment:
This isn't related to the PR, but the parentPID wasn't used anymore, so I removed it.